test/evergreen.yml

```diff
@@ -2752,6 +2897,14 @@ buildvariants:
     - name: long-test
     - name: static-wt-build-test
     - name: format-failure-configs-test
+    - name: data-validation-stress-test-checkpoint
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-1
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s2
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s3
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s4
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s5
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s6
+    - name: data-validation-stress-test-checkpoint-fp-hs-insert-s7
```

Comment:
As all these new tasks call `test/checkpoint` program under the hood, I suggest we line up the task names with the existing `checkpoint-test` task, to make it easier to search on Evergreen UI. For example, "checkpoint-test-stress-data-validation". Also, if these new tasks are categorized as stress tests, we have `Ubuntu 20.04 stress tests` builder that may suit them better?

Reply:
I'm also wondering about the benefits of splitting those tests into multiple Evergreen tasks. Normally we split to achieve a shorter turnaround time which is important for PR testing. Since there's no plan to add these new tasks into our PR testing scope, I'd be perfectly fine to just have a single new task to run all `-s` values in sequence.

Reply:
Hi @lukech , I created those multiple tasks, and I'm happy if someone wants to change them. An additional advantage of having them seperate is that if any single test fails, the others still run. That way an engineer can see if you get multiple failures occurring, and also see which ones still pass.

Reply:
Got your point @jeremythorp. It sounds like the target you'd like to achieve could be fulfilled by sub-tasks (in a single task), which are widely used in those server projects. We have not adopted the sub-task feature in WiredTiger projects yet.

Reply:
Hi @lukech , I'm not familiar with sub-tasks in Evergreen. If you have links to docs or examples I could look at that would be great.

Reply:
Here is an example - a server [evergreen task](https://evergreen.mongodb.com/task/mongodb_mongo_master_enterprise_rhel_80_64_bit_dynamic_required_display_concurrency_sharded_multi_stmt_txn_terminate_primary_f9383c046f2895f6622fe48f063ab3c174afcbb6_21_09_16_21_09_17) that includes multiple execution/sub-tasks. You can also check out the panel on the top-right corner of the page for the detailed tests of those sub-tasks. Considering we do not have past experience of adopting these features, and our tests are not that complicated to make use of them, I think we should keep using plain tasks.